src/ec/eddsa.rs

```diff
@@ -245,6 +245,24 @@ impl ExtPoint {
     }
 }
 
+// Keep this in sync with `ge_p2` in curve25519/internal.h.
+#[repr(C)]
+struct Point {
+    x: Elem,
+    y: Elem,
+    z: Elem,
+}
+
+impl Point {
+    fn new_at_infinity() -> Self {
```

Comment:
Of the current commits in this PR, this is the only one that won't build, because `Point` is dead code. So, let me know if you'd rather fold this into the next (verification) commit.